app/components/ui/Accordion/AccordionItem.jsx

```diff
@@ -2,33 +2,19 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 // TODO Enforcing the user to add their own dropdown with a custom renderer is a bit clunky
-class AccordionItem extends React.Component {
-  render() {
-    return (
-      <div>
-        <header>
-          {
-            this.props.headerRenderer
-              ? this.props.headerRenderer(this.props.title)
-              : (
-                <div>
-                  { this.title }
-                  <div className="selector">
-                    <i className="material-icons">keyboard_arrow_down</i>
-                  </div>
-                </div>
-              )
-          }
-        </header>
-        <section>{ this.props.children }</section>
-      </div>
-    );
-  }
-}
+const AccordionItem = ({ children, headerRenderer, title }) => (
+  <div>
+    <header>
+      {headerRenderer(title)}
```

Comment:
Should this be null-checked still (or the headerRenderer set as an `isRequired` prop type)?

Reply:
I thought the same thing at first, but actually it looks like there's a `defaultProp` for this property that renders just the title as a string.